streampark-common/src/main/scala/org/apache/streampark/common/conf/Workspace.scala

```diff
@@ -102,6 +102,11 @@ case class Workspace(storageType: StorageType) {
    */
   lazy val APP_JARS = s"$WORKSPACE/jars"
 
+  /**
+   * project archives path
+   */
+  lazy val ARCHIVES_FILE_PATH = s"$WORKSPACE/historyserver/archive"
```

Comment:
lazy val ARCHIVES_FILE_PATH = s"${Workspace.remote.WORKSPACE}/historyserver/archive"

Reply:
This is how I call it ：Workspace.remote().ARCHIVES_FILE_PATH()

Reply:
> lazy val ARCHIVES_FILE_PATH = s"${Workspace.remote.WORKSPACE}/historyserver/archive"

The design of this class is flawed and I'm working on re-improving it

Reply:
> lazy val ARCHIVES_FILE_PATH = s"${Workspace.remote.WORKSPACE}/historyserver/archive"

see: https://github.com/apache/incubator-streampark/pull/1928